loops/perl/code.pl

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/perl
```

Comment:
Maybe add

```perl
use strict;
use warnings;
```

here too?

Reply:
Didn't see the need. There's a pointless warning for uninitialized variables. I could ignore it specifically, but meh.